src/v/coproc/router.cc

```diff
@@ -118,39 +138,31 @@ ss::future<> router::route() {
 }
 
 ss::future<> router::do_route() {
-    auto reducer =
```

Comment:
was this code added in the previous patch and now being moved around in the next patch?